dataeng/resources/video-timeline.sh

```diff
@@ -0,0 +1,8 @@
+if [ "x$NUM_REDUCE_TASKS" = "x" ]; then
```

Comment:
I would recommend adding a shebang here. Unless explicitly told, Jenkins default shell env is pulled from the path, and in certain cases, that is dash, instead of bash (or even sh), depending on how you configured your system.